spinq10q_12_zcu111/platform.py

```diff
@@ -0,0 +1,94 @@
+import pathlib
+
+from qibolab.channels import Channel, ChannelMap
+from qibolab.instruments.rfsoc import RFSoC
+from qibolab.instruments.rohde_schwarz import SGS100A
+from qibolab.platform import Platform
+from qibolab.serialize import (
+    load_instrument_settings,
+    load_qubits,
+    load_runcard,
+    load_settings,
+)
+
+# from qibolab.kernels import Kernels
+
+NAME = "spinq10q_12__zcu111"
```

Comment:
```suggestion
NAME = "spinq10q_12_zcu111"
```

Just to match the name of the folder (there was an underscore in excess)

Reply:
I would even remove
```suggestion
```
I think this variable is not used anywhere and is even duplicating the `FOLDER` defined in line 21.